packages/fxa-auth-server/lib/routes/subscriptions/paypal-notifications.ts

```diff
@@ -56,15 +59,7 @@ export class PayPalNotificationHandler extends PayPalHandler {
             message: 'No idempotency key on PayPal transaction',
           });
         }
-        const { paymentAttempt } = this.paypalHelper.parseIdempotencyKey(
-          message.custom
```

Comment:
What is `message.custom`?

Reply:
PayPal provides a `CUSTOM` field that can be used by the developer as desired and is retained (but not shown to the user).